lib/utils/index.js

```diff
@@ -1,13 +1,13 @@
-import getCurrentItem from './getCurrentItem';
-import getCurrentList from './getCurrentList';
-import getDeepestItemDepth from './getDeepestItemDepth';
-import getItemDepth from './getItemDepth';
-import getItemsAtRange from './getItemsAtRange';
-import getListForItem from './getListForItem';
-import getPreviousItem from './getPreviousItem';
-import isList from './isList';
-import isItem from './isItem';
-import isSelectionInList from './isSelectionInList';
+import { getCurrentItem } from './getCurrentItem';
+import { getCurrentList } from './getCurrentList';
+import { getDeepestItemDepth } from './getDeepestItemDepth';
+import { getItemDepth } from './getItemDepth';
+import { getItemsAtRange } from './getItemsAtRange';
+import { getListForItem } from './getListForItem';
+import { getPreviousItem } from './getPreviousItem';
+import { isList } from './isList';
+import { isItem } from './isItem';
+import { isSelectionInList } from './isSelectionInList';
```

Comment:
```suggestion
import isSelectionInList from './isSelectionInList';
```
There is a practice of exporting the function by default when the name matches with filename.

Reply:
I've done this as a preparation for `export *` syntax, I just didn't know it was possible to use it in this repo... I've replaced all these import/exports with `export *`.